.gitignore

```diff
@@ -5,4 +5,4 @@ clover.xml
 
 # Commit your application's lock file https://getcomposer.org/doc/01-basic-usage.md#commit-your-composer-lock-file-to-version-control
 # You may choose to ignore a library lock file http://getcomposer.org/doc/02-libraries.md#lock-file
-# composer.lock
```

Comment:
Can you reset the composer.lock file. This way, our phpstan version and stuff stay in a "managed" version.